DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -248,6 +248,24 @@ def __update_field(self, field, value):
         field.Auto = False
         field.update(**param)
 
+    def __update_frame_rate(self, config_element, pkt_data):
+        """
+        Update frame rate type and frame rate from pkt_data
+
+        Args:
+            config_element (ConfigElement): grouping of endpoints under the Traffic Item per unique packet structure
+            pkt_data (dict): Packet stream config
+        """
+        frame_rate_types = {
+            "line_rate": "percentLineRate",
+            "bps_rate": "bitsPerSecond",
+            "ipg_rate": "interPacketGap",
```

Comment:
IPG is not a rate. You can use it in combination with any rate setting.

Reply:
IXIA rest.py defines it as a separate `rate_type `, therefore added here.